src/libostree/ostree-repo-commit.c

```diff
@@ -308,16 +308,19 @@ commit_loose_object_trusted (OstreeRepo        *self,
 
 typedef struct
 {
-  gsize unpacked;
-  gsize archived;
+  OstreeObjectType objtype;
+  guint64 unpacked;
+  guint64 archived;
```

Comment:
You made these 64 bit, but all the entry functions are still using `gsize`. Was that intentional?